packages/core/stories/state-icon.doc.stories.mdx

```diff
@@ -0,0 +1,68 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+import { ComponentAnatomy } from "docs/components/ComponentAnatomy";
+import { Button, StateIcon } from "@jpmorganchase/uitk-core";
+import {
+  DownloadIcon,
+  SearchIcon,
+  SendIcon,
+  SettingsSolidIcon,
+} from "@jpmorganchase/uitk-icons";
+import { withFlexGap } from "docs/decorators/withFlexGap";
+import { CustomStylingExample } from "./button.stories";
+import {
+  CSSClassTable,
+  CSSVariableTable,
+  CharacteristicUsage,
+} from "css-variable-docgen-components";
+
+<Meta
+  title="Documentation/Core/StateIcon"
+  component={StateIcon}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# State icon
+
+A state icon can be used on its own or within another component (e.g. `Tooltip`) to help convey a message.
+
+<Canvas>
+  <Story id="core-stateicon--default-state-icon" />
+</Canvas>
+
+## How you can use state icon
+
+Use this component by passing in your chosen `state` from the available options.
+
+You can also customise the icon's `size` and pass custom props to the unterlying svg element via `SVGProps`.
+
+# API
+
+```
+import { StateIcon } from "@jpmorganchase/uitk-core";
+```
+
+## Props
+
+<ArgsTable of={StateIcon} />
```

Comment:
`size` has a default value from `Icon`, not sure why it's not shown on the props table default column

Reply:
Yeah it's very odd, I'm looking into this

Reply:
I may not be possible BTW, doc gen may not be able to trace through at all. If so, may be set the default value in state icon as well.

Reply:
It doesn't seem to be possible unfortunately, I've tried a few different ways but this seems to be a known issue in Storybook. I've exported the default `size` from `Icon` and added it to `StateIcon`